config.c

```diff
@@ -182,6 +184,9 @@ static bool apply_config_option(struct mako_config *config, const char *section,
 		return parse_int(value, &config->max_visible);
 	} else if (strcmp(name, "default-timeout") == 0) {
 		return parse_int(value, &config->default_timeout);
+	} else if (strcmp(name, "ignore-timeout") == 0) {
+		config->ignore_timeout = true;
```

Comment:
Because this can be specified in the config, we should probably parse the value too as a bool (just like `markup`). Also, in the future this will allow things like:

```ini
ignore-timeout=1

[priority=urgent]
ignore-timeout=0
```